uwblueprint/jump-math
# (6) feat: edit classroom from drilldown

## Notion ticket link

[Edit Classroom from Class page](https://www.notion.so/uwblueprintexecs/Edit-Classroom-from-Class-page-847fc8f0937344958188616650234b36)

_Reviewing by commit might make the PR easier to review!_


## Implementation description
- [feat: update classroom mutation](https://github.com/uwblueprint/jump-math/pull/428/commits/d3285e49601074834630b566b6f7fe085e3455ec)
- [feat: add classroom-editing modal](https://github.com/uwblueprint/jump-math/pull/428/commits/f0b7dfa0b2084ebc86600be4ead28afb3b445c97)
- [chore: rename modal to be more accurate](https://github.com/uwblueprint/jump-math/pull/428/commits/f70dad799e4b3d30f841cb2683e8ecb4b6394de7)

<img width="1384" alt="image" src="https://github.com/uwblueprint/jump-math/assets/9922514/9f68c5b8-f418-4feb-bf09-d2eadbebed92">




## Steps to test
1. View the class list page.
2. Click on a class card.
3. Click the pencil icon next to the classroom title.
4. Verify the modal that opens can properly update all the classroom data.



## What should reviewers focus on?
* Is the code clean enough?
* Are there edge cases I missed?


## Checklist
